riotfile.py

```diff
@@ -2493,29 +2493,19 @@ def select_pys(min_version: str = MIN_PYTHON_VERSION, max_version: str = MAX_PYT
             },
             venvs=[
                 Venv(
-                    # openai[embeddings] broken install with sklearn was never fixed on 0.26
-                    # https://github.com/openai/openai-python/issues/210
-                    pys="3.7",
-                    env={"SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL": "True"},
-                    pkgs={
-                        "openai": "==0.26.5",
-                        "scikit-learn": "==1.0.2",
-                        "pillow": "==9.5.0",
-                    },
-                ),
-                Venv(
-                    pys=select_pys(min_version="3.7", max_version="3.11"),
+                    pys=select_pys(min_version="3.8", max_version="3.11"),
```

Comment:
```suggestion
                    pys=select_pys(min_version="3.7", max_version="3.11"),
```
let's keep 3.7 to avoid regressions for ddtrace 2.x